plugins/notifications-backend/src/database/DatabaseNotificationsStore.test.ts

```diff
@@ -288,6 +299,164 @@ describe.each(databases.eachSupportedId())(
+        });
+        await insertNotification({
+          id: id3,
+          user,
+          origin: 'X',
+          payload: {
+            title: 'Notification 3',
+            link: '/catalog',
+            severity: 'normal',
+            topic: 'BBB',
+          },
+          created: new Date(now - 5 * timeDelay),
+        });
+
+        const notificationsCreatedAsc = await storage.getNotifications({
+          user,
+          sort: 'created',
+          sortOrder: 'asc',
+        });
+        expect(notificationsCreatedAsc.length).toBe(3);
+        expect(notificationsCreatedAsc.at(0)?.id).toEqual(id1);
+        expect(notificationsCreatedAsc.at(1)?.id).toEqual(id3);
+        expect(notificationsCreatedAsc.at(2)?.id).toEqual(id2);
+
+        const notificationsCreatedDesc = await storage.getNotifications({
+          user,
+          sort: 'created',
+          sortOrder: 'desc',
+        });
+        expect(notificationsCreatedDesc.length).toBe(3);
+        expect(notificationsCreatedDesc.at(0)?.id).toEqual(id2);
+        expect(notificationsCreatedDesc.at(1)?.id).toEqual(id3);
+        expect(notificationsCreatedDesc.at(2)?.id).toEqual(id1);
+
+        const notificationsTopicAsc = await storage.getNotifications({
+          user,
+          sort: 'topic',
+          sortOrder: 'asc',
+        });
+        expect(notificationsTopicAsc.length).toBe(3);
+        expect(notificationsTopicAsc.at(0)?.id).toEqual(id1);
+        expect(notificationsTopicAsc.at(1)?.id).toEqual(id3);
+        expect(notificationsTopicAsc.at(2)?.id).toEqual(id2);
+
+        const notificationsTopicDesc = await storage.getNotifications({
+          user,
+          sort: 'topic',
+          sortOrder: 'desc',
+        });
+        expect(notificationsTopicDesc.length).toBe(3);
+        expect(notificationsTopicDesc.at(0)?.id).toEqual(id2);
+        expect(notificationsTopicDesc.at(1)?.id).toEqual(id3);
+        expect(notificationsTopicDesc.at(2)?.id).toEqual(id1);
+
+        const notificationsOrigin = await storage.getNotifications({
+          user,
+          sort: 'origin',
+          sortOrder: 'asc',
+          limit: 2,
+          offset: 0,
+        });
+        expect(notificationsOrigin.length).toBe(2);
+        expect(notificationsOrigin.at(0)?.id).toEqual(id3);
+        expect(notificationsOrigin.at(1)?.id).toEqual(id1);
+
+        const notificationsOriginNext = await storage.getNotifications({
+          user,
+          sort: 'origin',
+          sortOrder: 'asc',
+          limit: 2,
+          offset: 2,
+        });
+        expect(notificationsOriginNext.length).toBe(1);
+        expect(notificationsOriginNext.at(0)?.id).toEqual(id2);
+      });
+
+      it('filters on severity', async () => {
+        const severities: (NotificationSeverity | undefined)[] = [
+          'normal',
+          undefined,
+          'critical',
+          'high',
+          'low',
+        ];
+        const now = Date.now();
+        await Promise.all(
```

Comment:
Again this suite of tests should be wrapped in a `describe` and this should be moved to `beforeAll`. I would similarly clean up the tests as outlined above.